src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -84,6 +84,12 @@ internal void OnMessageLogged(ILogMessage msg)
         public void Loaded(ViewLoadedParams viewLoadedParams)
         {
             DependencyView = new WorkspaceDependencyView(this, viewLoadedParams);
+            // when a package is loaded update the DependencyView 
+            // as we may have installed a missing package.
+            pmExtension.PackageLoader.PackgeLoaded += (package) =>
+            {
+                DependencyView.DependencyRegen(viewLoadedParams.CurrentWorkspaceModel as WorkspaceModel);
```

Comment:
I guess this way, if a package is downloaded using the package manager interface, the view extension here will pick up that change as well? Does that cover all the cases in Dynamo?

Reply:
@QilongTang it doesn't cover other dependency types - I'm also not sure if it covers packages being uninstalled - do you want me to add this same handler for package removal? (if that event exists?)

Reply:
it does exist ` pmExtension.PackageLoader.PackageRemoved`

Reply:
a bit hesitant to get into packageRemoval now and how it relates to package's being in a pending uninstall state.
Would like to defer adding this handler until later or atleast splitting this into two PRs.

Reply:
Let's leave the uninstall case for now, I would like to give it some testing first